lib/frontman/commands/serve.rb

```diff
@@ -83,8 +84,34 @@ def serve
   end
 end
 
-class FrontManServer < Sinatra::Base
-  set :port, 4568
+class FrontmanServer < Sinatra::Base
+  port = Frontman::Config.get(:port, fallback: 4568)
+  num_retries = Frontman::Config.get(:port_retries, fallback: 3)
+
+  port_retry_strategy = Frontman::Config.get(:port_retry_strategy, ->(port_num){
```

Comment:
```suggestion
  port_retry_strategy = Frontman::Config.get(:port_retry_strategy, fallback: ->(port_num){
```

Reply:
We should increase the MaxLineLength to 100 chars. I believe it often allows for more meaningful variable naming.

I had to shorten to `retry_strategy = Frontman::Config.get(:port_retry_strategy, fallback: ->(p) {` and that just barely gets me under the 80 char limit.

Reply:
We can change the formatting of the code a bit, like this:

```ruby
port_retry_strategy = Frontman::Config.get(
    :port_retry_strategy, 
    fallback: ->(port_num) {
        # retry logic
    }
)
```

This way, we don't have to change the `MaxLineLength`, and still have the meaningful names. WDYT?

Reply:
Yes sure we could do that.

I just thought it would be a really nice improvement to the development experience by giving developers a little more room to be slightly more descriptive and concise. In my opinion, 80 characters just seems so limiting and 100 seems to be much more reasonable to work within.